content/applications/sales/crm/performance/win_loss.rst

```diff
@@ -1,26 +1,85 @@
-=========================
-Check your Win/Loss Ratio
-=========================
+========================
```

Comment:
Thanks for changing the branch name to CRM! Reminder to change the name of the PR to 
[IMP] CRM: ... too, but clicking the edit button at the top of the PR 
![image](https://github.com/odoo/documentation/assets/23611711/02ffa5ce-dd64-4ad9-ae7e-ed66a4208af6)